index.bs

```diff
@@ -632,19 +632,281 @@ Issue(12): Propose control protocol for Remote
 Playback API.
 
 
-Security and Privacy {#security}
-================================
-
-Issue(13): Describe security architecture.
+Security and Privacy {#security-privacy}
+====================
 
-Data to be secured {#security-data}
+The Open Screen Protocol allows two networked user agents to discover each other
+and exchange user and application data.  As such, its security and privacy
+considerations should be closely examined.  We first evaluate the protocol
+itself using the W3C [[SECURITY-PRIVACY-QUESTIONNAIRE|Security and Privacy
+Questionnaire]].  We then examine whether the security and privacy guidelines
+recommended by the [[PRESENTATION-API|Presentation API]] and the
+[[REMOTE-PLAYBACK|Remote Playback API]] are met.  Finally we discuss recommended
+mitigations that implementations can use to meet these security and privacy
+requirements.
+
+Threat Models {#threat-models}
+--------------------------------
+
+### Passive Network Attackers ### {#passive-network-attackers}
+
+The Open Screen Protocol should assume that all parties that are connected to
+the same LAN, either through a wired connection or through WiFi, are able to
+observe all data flowing between Open Screen controllers and receivers.
```

Comment:
perhaps just "between agents".  We might have more than controllers and receivers in the future.

Reply:
Done.